src/lib.rs

```diff
@@ -572,6 +546,33 @@ impl TestKit {
         self.poll_events();
     }
 
+    // Try to modify the test network configuration.
+    fn do_config_update(&mut self) {
+        let height = self.height();
+        if let Some(cfg_proposal) = self.cfg_proposal.take() {
+            match cfg_proposal {
+                ConfigurationProposalState::Uncommited(cfg_proposal) => {
+                    // Commit configuration proposal
+                    let stored = cfg_proposal.stored_configuration().clone();
+                    let mut fork = self.blockchain.fork();
+                    CoreSchema::new(&mut fork).commit_configuration(stored);
+                    let changes = fork.into_patch();
+                    self.blockchain.merge(changes).unwrap();
+                    self.cfg_proposal = Some(ConfigurationProposalState::Commited(cfg_proposal));
+                }
+                ConfigurationProposalState::Commited(ref cfg_proposal)
+                    if cfg_proposal.actual_from() == height => {
```

Comment:
This is one ugly formatting (automatic, I suppose) 😕

Reply:
Fixed